OpenSim/Simulation/Manager/Manager.h

```diff
@@ -95,38 +95,29 @@ class OSIMSIMULATION_API Manager
     double _ti;
     /** Final time of the simulation. */
     double _tf;
-    /** First dt in an integration. */
-    double _firstDT;
     
     /** Storage for the states. */
     std::unique_ptr<Storage> _stateStore;
 
-   int _steps;
-   /** Number of integration step tries. */
-   int _trys;
-   /** Maximum number of steps in an integration. */
-   int _maxSteps;
-   /** Flag for signaling a desired halt. */
-   bool _halt;
-   /** Minimum step size. */
-   double _dtMin;
```

Comment:
> Oddly enough, ForwardTool does respect _maxDT, so I think this is just an oversight.

 I think so too.

> should AnalyzeTool derive from AbstractTool? 

That's a bigger issue. I think it was wrong to have integrator settings as `AbstractTool` properties. They should have been properties  of the `Manager` that creates the integrator. Then any `Tool` that has a `Manager` property would have the integrator settings where/when it needs it. This is beyond the scope of this PR so leave it for the time being.